packages/instantsearch.js/src/connectors/toggle-refinement/connectToggleRefinement.ts

```diff
@@ -425,6 +426,19 @@ const connectToggleRefinement: ToggleRefinementConnector =
         },
 
         getWidgetSearchParameters(searchParameters, { uiState }) {
+          if (
+            searchParameters.isHierarchicalFacet(attribute) ||
+            searchParameters.isConjunctiveFacet(attribute)
+          ) {
```

Comment:
Nitpick, but why do you have a condition as a wrapper instead of putting the negated assertion in the first parameter of `warning()`?

Reply:
🤦  That explains it, thank you!